manifest-form-runner.yml

```diff
@@ -8,9 +8,11 @@ applications:
   routes:
     - route: forms.dev.gids.dev
   env:
-    PREVIEW_MODE: TRUE
+    NODE_ENV: production
     RSA256_PUBLIC_KEY_BASE64: ((RSA256_PUBLIC_KEY_BASE64))
     SERVICE_START_PAGE: https://frontend.dev.gids.dev/account
+    JWT_REDIRECT_TO_AUTHENTICATION_URL: https://authenticator2.dev.gids.dev/sessions/sign-out
+    SAFELIST: '["funding-service-design-application-store-dev.london.cloudapps.digital"]'
```

Comment:
Don't think this will work as it's not defined here: https://github.com/communitiesuk/digital-form-builder/blob/main/runner/config/custom-environment-variables.json ?
You might need to leave it in `NODE_CONFIG` for now.

Reply:
Reverted to having this defined in copied in [production.json](https://github.com/communitiesuk/funding-service-design-frontend/pull/106/files#diff-45ba9790f0e88dec4753d77532166925ad4337c545d91b90db6a429b779d53c7)